test/katex-spec.js

```diff
@@ -912,6 +912,43 @@ describe("A color parser", function() {
     });
 });
 
+describe("Alpha hex color parser", function() {
+    const alphaColorExpression1 = r`\textcolor{#ff000080}{x}`;
+    const alphaColorExpression2 = r`\textcolor{228B22BF}{y}`;
+    const alphaColorExpression3 = r`\textcolor{#1234ABCD}{z}`;
+    const badAlphaColorExpression1 = r`\textcolor{#ff00008g}{x}`;
+    const badAlphaColorExpression2 = r`\textcolor{#ff00008}{x}`;
+    const badAlphaColorExpression3 = r`\textcolor{#ff000080f}{x}`;
+
+    it("should parse 8-digit hex colors with alpha", function() {
+        expect(alphaColorExpression1).toParse();
+        expect(alphaColorExpression2).toParse();
+        expect(alphaColorExpression3).toParse();
+    });
```

Comment:
These tests can be omitted. You're already testing parsing below, and building (which includes parsing) later.